packages/valory/skills/check_stop_trading_abci/models.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2024 Valory AG
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+
+
+"""Models for the check stop trading ABCI application."""
+
+from typing import Any
+
+from packages.valory.skills.abstract_round_abci.models import BaseParams
+from packages.valory.skills.abstract_round_abci.models import (
+    BenchmarkTool as BaseBenchmarkTool,
+)
+from packages.valory.skills.abstract_round_abci.models import Requests as BaseRequests
+from packages.valory.skills.abstract_round_abci.models import (
+    SharedState as BaseSharedState,
+)
+from packages.valory.skills.check_stop_trading_abci.rounds import CheckStopTradingAbciApp
+
+
+Requests = BaseRequests
+BenchmarkTool = BaseBenchmarkTool
+
+
+class CheckStopTradingParams(BaseParams):
+    """CheckStopTrading parameters."""
+
+    staking_contract_address: str
+    mech_contract_address: str
```

Comment:
These are required because they are defined in another skill's params

Reply:
Can you explain why not override the params of the staking skill instead?

Reply:
Fixed. The reasoning was that I thought it was a bit uglier how the Params need to be loader on the composed app now:

```
class TraderParams(
    DecisionMakerParams,  # It also contains MechInteractParams
    TerminationParams,
    TxSettlementMultiplexerParams,
    CheckStopTradingParams,  # It also contains StakingParams
):
```

... but I noticed it already happens for `MechInteractParams`